docs/source/cluster_management/sweep/sweep-endpoints.rst

```diff
@@ -5,24 +5,20 @@ AtlasDB Sweep Endpoints
- +------------------------------------------+-------------------------------------------------+--------------------------------------------------------------------------------------------------+
- |``/sweep/sweep-table-from-row-with-batch``| ``tablename``: fully qualified table name       | Sweep a particular table from specified startRow with specified ``SweepBatchConfig`` parameters. |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``startRow``: base16 encoded start row          |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``maxCellTsPairsToExamine``: cells to examine   |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``candidateBatchSize``: cells to read per batch |                                                                                                  |
- +                                          +-------------------------------------------------+                                                                                                  |
- |                                          | ``deleteBatchSize``: cells to delete per batch  |                                                                                                  |
- +------------------------------------------+-------------------------------------------------+--------------------------------------------------------------------------------------------------+
+ +-----------------------+-------------------------------------------------+---------------------------+---------------------------------------------------------------------------------+
+ | Endpoint              | Query Parameters                                | Required/Optional         | Description                                                                	 |
+ +=======================+=================================================+===========================+=================================================================================+
+ |``/sweep/sweep-table`` | ``tablename``: fully qualified table name       | Required                  |Sweep a particular table from EMPTY startRow with default ``SweepBatchConfig``.  |
```

Comment:
We should probably clarify that we sweep from `startRow` with the behavior specified by the other parameters.